docs/tutorials/experiment_cloud_service.ipynb

```diff
@@ -232,7 +232,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Load an experiment from the database"
+    "### Loading an experiment from the database"
```

Comment:
For this tutorial, the title and `T1 Experiment`, `RB Experiment`, and `State Tomography Experiment` need to be converted to regular capitalization. `Auto save an experiment` also needs to be converted to present progressive